scripts/rollup_build_hook.py

```diff
@@ -1,9 +1,21 @@
+import pathlib
+import shutil
 import subprocess
 
 from hatchling.builders.hooks.plugin.interface import BuildHookInterface
 
 
 class RollupBuildHook(BuildHookInterface):
     def initialize(self, version, build_data):
+        path = (
+            pathlib.Path(__file__).parent
```

Comment:
Adding ".." is the same as adding another `.parent`. Please do `.parent` instead.
Also, the resolve should come first, not last:
`pathlib.Path(__file__).resolve().parent.parent ...`